### ISSUE TITLE
Feature Request:  subnetContains(subnet, ip) -> bool

### ISSUE THREAD

**Issue by donspaulding** [indutny/node-ip#18]
Given a subnet and an IP address, I'd like to know if the address falls within the usable addresses of the subnet.  A rough draft of the API I'm interested in would look something like this:
```javascript
ip.subnetContains('192.168.1.1/24', '192.168.1.14');
// true
ip.subnetContains('192.168.0.1/24', '192.168.1.14');
// false
````

**Comment by mkleehammer** [comment]
Any progress on this?

**Comment by leachiM2k** [comment]
I would like to see this as well!

**Comment by bryanwayb** [comment]
This has just been added and is available in version 1.1.0
